docs/devops.md

```diff
@@ -10,7 +10,334 @@ The stack is managed using the following tools:
+7. Log into the deploy profile:
+    - `aws sso login --profile bceln-deploy`
+8. Provision the node with Ansible: 
+    - `ansible-playbook microk8s.yml -i inventory/prod --diff --ask-become-pass -l new.node.hostname` where `new.node.hostname` is, for example, `beln-arca-oc.dc.sfu.ca`
+9. Shell into the `arca-dc` server, and add the node to the cluster:
+    - `microk8s add-node`
+    - Copy the `microk8s join` command it generates.
+10. Shell into the new server, and paste the `microk8s join` command from the previous step.
+    - If there's a timeout error, there is probably a firewall issue between the servers; contact SFU to deal with it.
+11. On the `arca-dc` server, confirm the node has been added with `kubectl get nodes`.
+
+
+## Install the new site
+
+1. Shell into the Douglas College server
+    - `ssh beln-arca-dc.dc.sfu.ca`
+2. Go to to the `helm_values` directory
+    - `cd /opt/helm_values`
+3. Copy an existing site's directory tree for your new site.
+    - e.g. if you are creating a new site, "oc", copy the "dc" directory tree:
+    - `cp -r dc oc`
+4. Edit the deployment configuration files:
+    - `shared` directory: `cd /opt/helm_values/[newsite]/shared`
+        - [QUESTION: "The configurations in this directory are used by multiple services by being symlinked." Does this mean changing it here changes it everywhere? It doesn't look like it does, as the path from `dc` and the path from `oc` show different values.]
+        - Edit `affinity.yaml` to set the new site's hostname
+            - eg. ```---
+                     nodeSelector:
+                     # Set to the hostname of the node the site should run on
+                     kubernetes.io/hostname: beln-arca-oc.dc.sfu.ca
+                  ```
+        - Edit `ingress.yaml` with the public hostname of the site (that is, the URL it will be accessed from).
+            - If migrating an existing site, you may put both the temporary URL and the final production URL here. 
+            - e.g.
+              ```
+                ingress:
+                  # Set the hostname for the site
+               - host: oc.arcabc.ca
+                 tls:
+                     issuer: letsencrypt-prod
+              - host: tempname.arcabc.ca
+                 tls:
+                     issuer: letsencrypt-prod
+              ```
+    - `drupal` directory: `cd /opt/helm_values/[newsite]/drupal`
+        - Edit `values.yaml`:
+            - `FEDORA_OBJECT_PATH`: For migration; points at mounted path of the objectStore for the site. Not necessary if not migrating.
+            - `image: tag`: Sets the version of the Drupal image that will be deployed
+                - Usually it will be the [latest tag in the bceln-drupal repo](https://github.com/discoverygarden/bceln-drupal/tags) without the v.
+        - IF MIGRATING: Do not change `base.yaml` as it's common data across the whole system
+            - IF NOT MIGRATING: Remove the lines relating to Fedora.
```

Comment:
When not migrating it should be safe to be left in place if access to the mount is requested on all server.